src/List/Selection.elm

```diff
@@ -115,3 +151,43 @@ map fn (Selection selected items) =
     Selection
         (Maybe.map fn selected)
         (List.map fn items)
+
+
+{-| Apply a function to only the selected item, if any item is selected.
+-}
+mapSelected : (a -> a) -> Selection a -> Selection a
+mapSelected fn (Selection selected items) =
+    Selection
+        (Maybe.map fn selected)
+        items
```

Comment:
🐛 Doesn't this break  `toListWithSelected` because it's not `==` anymore?

Reply:
and `toList` wouldn't return the mapped item. It would still be the original value.